src/runtime_src/core/tools/xbutil2/SubCmdReset.cpp

```diff
@@ -80,14 +113,14 @@ SubCmdReset::execute(const SubCmdOptions& _options) const
   XBU::verbose("SubCommand: reset");
   // -- Retrieve and parse the subcommand options -----------------------------
   std::vector<std::string> devices = {"all"};
-  std::string resetType = "hot"; //default val
+  std::string resetType = "user"; //default val
```

Comment:
Comment here isn't needed. 

If one really wanted to comment that these values are being initialized to default values, then 1 comment prior to devices would be best.  Something like:
`// Initializing variables to their default values`

Reply:
Done